fdb-record-layer-core/src/test/java/com/apple/foundationdb/record/provider/foundationdb/OnlineIndexerSimpleTest.java

```diff
@@ -737,10 +472,7 @@ public void testOnlineIndexerBuilderWriteLimitBytes() throws Exception {
                                  .setMaxWriteLimitBytes(1)
                                  .build()) {
 
```

Comment:
With this change, what is the difference between this block, and the block above (currently lie 699/434)

Reply:
The second block is called after `recordStore.clearAndMarkIndexWriteOnly(...)`, which forces it to write more data. But I will get rid from the first one. 
✅

Reply:
The first block was doing `rebuildIndex` and checking that the metrics were right. This one was calling `buildUnbuiltRange`, and we're removing the call to `buildUnbuiltRange`